test/unit_tests/graphql/test_processes.py

```diff
@@ -89,19 +96,27 @@ def get_processes_query_with_subscriptions(
 query ProcessQuery($first: Int!, $after: Int!, $sortBy: [GraphqlSort!], $filterBy: [GraphqlFilter!]) {
   processes(first: $first, after: $after, sortBy: $sortBy, filterBy: $filterBy) {
     page {
-      assignee
-      createdBy
-      failedReason
+      processId
       isTask
       lastStep
+      lastStatus
+      assignee
+      failedReason
       traceback
-      processId
-      lastModified
-      started
       workflowName
-      status
-      step
-      product
+      createdBy
+      startedAt
+      lastModifiedAt
+      product {
+        productId
```

Comment:
Can't we just have id (instead of product_id, subscription_id, process_id, etc.)? For the same reason we also don't use product_name, product_description, ...

Reply:
It is how it is generated from the pydantic models and I don't think we should specifically change it within graphql.